project/Dependencies.scala

```diff
@@ -42,7 +42,7 @@ object Dependencies {
   val cachecontrolVersion = "1.1.3"
   val cachecontrol = Seq("com.typesafe.play" %% "cachecontrol" % cachecontrolVersion)
 
-  val asyncHttpClientVersion = "2.0.39"
+  val asyncHttpClientVersion = "2.4.4"
```

Comment:
There is a 2.4.5 already: http://mvnrepository.com/artifact/org.asynchttpclient/async-http-client/2.4.5

Reply:
and its 2.4.6 now.. http://mvnrepository.com/artifact/org.asynchttpclient/async-http-client/2.4.6

Reply:
unfortunately there is a braking change in 2.4.5 https://github.com/AsyncHttpClient/async-http-client/commit/b819481deac6d01c0bf1d824a81085aa8b722110 
Couldn't find a way to fix it. Feel free to give a if you're familiar with play-ws build lifecycle